src/Command/Resources/ResourcesSetCommand.php

```diff
@@ -50,7 +50,9 @@ protected function configure()
             '',
             sprintf('Profile sizes are predefined CPU & memory values that can be viewed by running: <info>%s resources:sizes</info>', $this->config()->get('application.executable')),
             '',
-            'If the same service and resource is specified on the command line multiple times, only the final value will be used.'
+            'If the same service and resource is specified on the command line multiple times, only the final value will be used.',
+            '',
+            'You can also configure your apps to automatically scale using the <info>autoscaling:set</info> command.'
```

Comment:
This is good, but elsewhere, recommended commands are usually prefixed with `:` and the "executable" name, like the mention of `resources:sizes` just above-

```suggestion
            sprintf('You can also configure autoscaling, by running: <info>%s autoscaling:set</info>.', $this->config()->get('application.executable')),
```